Modules/Core/ImageFunction/include/itkGaussianBlurImageFunction.hxx

```diff
@@ -78,22 +78,10 @@ template <typename TInputImage, typename TOutput>
 void
 GaussianBlurImageFunction<TInputImage, TOutput>::SetSigma(const double * sigma)
 {
-  unsigned int i;
-
-  for (i = 0; i < Self::ImageDimension; ++i)
+  if (ContainerCopyWithCheck(m_Sigma, sigma, Self::ImageDimension))
```

Comment:
How did you identify these instances? Local search for `this->Modified()` has ~1k hits. Most of them are scalar variables, but this might be a good way to identify the vector ones which would benefit from this refactoring.

Reply:
I only looked for the setter functions that had this particular signature where the ContainerCopyWithCheck and ContainerFillWithCheck could be used as a replacement.

Manual effort. :(